articles/synapse-analytics/monitoring/how-to-monitor-using-azure-monitor.md

```diff
@@ -106,6 +106,10 @@ Here are the logs emitted by Azure Synapse Analytics workspaces:
 | SynapseIntegrationActivityRuns | IntegrationActivityRuns | Azure Synapse integration activity runs. |
 | SynapseIntegrationTriggerRuns | IntegrationTriggerRuns | Azure Synapse integration trigger runs. |
 
+   > [!NOTE]  
+   > The event **SynapseBuiltinSqlReqsEnded** is only emitted for queries that read data from storage, it will not be emitted for queries that only process metadata.
```

Comment:
```suggestion
   > The event **SynapseBuiltinSqlReqsEnded** is only emitted for queries that read data from storage. It will not be emitted for queries that only process metadata.
```